state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -163,6 +163,11 @@ pub mod pallet {
 		Egress,
 	}
 
+	pub struct AmountFeesWithheld<T: Config<I>, I: 'static> {
```

Comment:
`AmountFeesWithheld` -> `AmountAndFeesWithheld`? Otherwise it is a bit misleading on the first read.

Reply:
I did this in the next PR targetting this: https://github.com/chainflip-io/chainflip-backend/pull/4458